simpeg_drivers/driver.py

```diff
@@ -401,7 +404,15 @@ def get_regularization(self):
                     )
 
                 if getattr(self.params, f"{comp}_norm") is not None:
-                    norms.append(getattr(self.params, f"{comp}_norm"))
+                    norm = getattr(self.params, f"{comp}_norm")
+                    if isinstance(norm, Data):
+                        norm = norm.values[self.models.active_cells]
+                        if comp in "xyz":
+                            norm = (
+                                getattr(reg.regularization_mesh, f"aveCC2F{comp}")
+                                * norm
+                            )
+                    norms.append(norm)
```

Comment:
Why not adding the norms to the `self.models`? Currently the norm.values is attached to the ordering of the input mesh, so if it is UBC it will likely get jumbled.